### PR TITLE
Add endpoints for getting member + inductee points

### PR DESCRIPTION
**Pull request by godwinpang**
RN this gives a 404 when you query for someone who doesn't have points which probably isn't the best but uh it works so idc for now :D

### CODE REVIEW

File: src/entities/Attendance.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class Attendance {
   @Column('timestamp', { nullable: true })
   endTime?: Date;
 
-  @Column({ nullable: true })
+  @Column({ type: 'float4', nullable: true })
```
Comment (thaigillespie) [inline src/entities/Attendance.ts:28]: This gives 6 decimal degits precision. Any reason why you picked this? Just curious.

File: src/controllers/UserController.ts [hunk 2]
```diff
@@ -117,6 +119,41 @@ export class UserController {
...
+    @CurrentUser({ required: true }) requestAppUser: AppUser
+  ): Promise<AppUserInducteePointsResponse | undefined> {
+    if (this.appUserService.isInvalidNonOfficerAccess(requestAppUser, userID)) {
+      throw new ForbiddenError();
+    }
+
+    const points = await this.appUserService.getInducteePoints(userID);
+
+    // I don't like this but I'll take it for now - Godwin Oct 24 2020
+    return points;
```
Comment (thaigillespie) [inline src/controllers/UserController.ts:138]: Perhaps renaming it to pointsObj or smth is better? There is a bit more information that just the points themselves.

File: src/controllers/UserController.ts [hunk 3]
```diff
@@ -117,6 +119,41 @@ export class UserController {
...
+    @Param('userID') userID: number,
+    @CurrentUser({ required: true }) requestAppUser: AppUser
+  ): Promise<AppUserMemberPointsResponse | undefined> {
+    if (this.appUserService.isInvalidNonOfficerAccess(requestAppUser, userID)) {
+      throw new ForbiddenError();
+    }
+
+    const points = await this.appUserService.getMemberPoints(userID);
+
+    return points;
```
Comment (thaigillespie) [inline src/controllers/UserController.ts:155]: Same as above for this

**Review by thaigillespie — CHANGES_REQUESTED** [review]
Lgtm but had some small things I wanted to bring up.

File: src/entities/Attendance.ts (hunk 1 above)
Comment (godwinpang) [reply src/entities/Attendance.ts:28]: it works :)

File: src/controllers/UserController.ts (hunk 2 above)
Comment (godwinpang) [reply src/controllers/UserController.ts:138]: idr why i did this don't ask :|

Comment (godwinpang) [reply src/controllers/UserController.ts:138]: I think points is fine for now... it's mainly just points right? idt pointsObj is strictly a better name, maybe induction requirements might be better lol

Comment (thaigillespie) [reply src/controllers/UserController.ts:138]: Yeah sounds good, I'll approve this then

**Review by thaigillespie — APPROVED** [review]
lgtm

**Pull request merged**